Vic2ToHoI4/Data_Files/configurables/cultureGroupToGraphics.txt

```diff
@@ -1812,8 +1812,163 @@ iranian = {
 	graphical_culture_2d = middle_eastern_2d
 }
 
-arabic_persian = {
-	cultures_and_groups = { arab punic iranian_turanian }
```

Comment:
Looks like persian, tajik, pashtun, baluchi, hazara, kurdish, sarmatian, tocharian, dinajidad, janub, saahi, and gypsy got dropped